user-manuals/User_Guide/Student/StudentGuide.md

```diff
@@ -0,0 +1,15 @@
+# User Guide for Student
```

Comment:
but this explanation is already covered in our wiki, I thought like explanation is not part of user guide, because in user guide its simply means like workflow, right? so this point is not part of user guide- "The skill levels are not mentioned at all (what do they mean? What affects them?)" @SkyDiverStar

Reply:
The stats page is not explained, the scoreboard is only mentioned in the small version, not the full scoreboard. for that we can include one screenshot for the scoreboard but explanation I guess is not part of user guide, because we already have concepts for that I guess @SkyDiverStar